doc/devices/qvm.rst

```diff
@@ -0,0 +1,224 @@
+
+.. math::
+    \langle Z \rangle_{i} = \frac{1}{N}\sum_{j=1}^N (1-2m_j)
+
+where :math:`N` is the total number of shots, and :math:`m_j` is the :math:`j` th measurement of wire :math:`i`.
+
+Change of measurement basis
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For the remaining observables, it is easy to perform a quantum change of basis operation before measurement such that the correct expectation value is performed. For example, say we have a unitary Hermitian observable :math:`\hat{A}`. Since, by definition, it must have eigenvalues :math:`\pm 1`, there will always exist a unitary matrix :math:`U` such that it satisfies the following similarity transform:
+
+.. math:: \hat{A} = U^\dagger Z U
+
+Since :math:`U` is unitary, it can be applied to the specified qubit before measurement in the Pauli-Z basis. Below is a table of the various change of basis operations performed implicitly by PennyLane.
+
++------------------+-----------------------------------+
+|    Observable    | Change of basis gate    :math:`U` |
++==================+===================================+
+| ``qml.PauliX``   | :math:`H`                         |
++------------------+-----------------------------------+
+| ``qml.PauliY``   | :math:`H S^{-1}=HSZ`              |
++------------------+-----------------------------------+
+| ``qml.Hadamard`` | :math:`R_y(-\pi/4)`               |
++------------------+-----------------------------------+
+
+To see how this affects the resultant quil program, you may use the :attr:`~.ForestDevice.program` property to print out the quil program after evaluation on the device.
+
+.. code-block:: python
+
+    dev = qml.device('forest.qvm', device='2q-qvm')
+
+    @qml.qnode(dev)
+    def circuit(x):
+        qml.RX(x, wires=[0])
+        return expval(qml.PauliY(0))
+
+>>> circuit(0.54)
+-0.525390625
+>>> print(dev.program)
+PRAGMA INITIAL_REWIRING "PARTIAL"
+RX(0.54000000000000004) 0
+Z 0
+S 0
+H 0
+DECLARE ro BIT[1]
+MEASURE 0 ro[0]
+
+.. note::
+
+    :attr:`~.ForestDevice.program` will return the **last evaluated quantum program** performed on the device. If viewing :attr:`~.ForestDevice.program` after evaluating a quantum gradient or performing an optimization, this may not match the user-defined QNode, as PennyLane automatically modifies the QNode to take into account the `parameter shift rule <https://pennylane.readthedocs.io/en/latest/concepts/autograd_quantum.html>`_, product rule, and chain rule.
```

Comment:
This link is also broken (old). Should it link to the glossary article instead?
```suggestion
    :attr:`~.ForestDevice.program` will return the **last evaluated quantum program** performed on the device. If viewing :attr:`~.ForestDevice.program` after evaluating a quantum gradient or performing an optimization, this may not match the user-defined QNode, as PennyLane automatically modifies the QNode to take into account the `parameter shift rule <https://pennylane.ai/qml/glossary/parameter_shift.html>`_, product rule, and chain rule.
```